assets/js/composer.js

```diff
@@ -343,12 +386,8 @@ PageComposer.prototype = {
     let $parentFormControl;
     let $positionFormControl;
 
-    applyAdmin($form);
```

Comment:
this applyAdmin was incorrect. it needs to be $childBlock and be called last on the line to be the same as edit. (now create gets disabled buttons until the ajax request is done)